jwst/lib/tests/test_set_telescope_pointing.py

```diff
@@ -211,7 +213,9 @@ def test_add_wcs_default(data_file):
             '\nException={}'.format(e)
         )
 
-    model = datamodels.open(data_file)
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", category=NoTypeWarning)
+        model = datamodels.open(data_file)
```

Comment:
An alternative for avoiding the warning is
```suggest
        model = datamodels.Level1bModel(data_file)
```

Reply:
I'm going to answer my own question.  Filtering warnings is more overhead than just opening as a `Level1bModel`.  So going for the latter.  The tests pass without warnings either way.